validphys2/src/validphys/pdfbases.py

```diff
@@ -683,59 +933,181 @@ def fitbasis_to_NN31IC(flav_info, fitbasis):
             matrix performing the change of basis from fitbasis to NN31IC
 
     """
-    if fitbasis == 'NN31IC':
+    if fitbasis == "NN31IC":
         return np.identity(8)
```

Comment:
If the output is to be 9 elements now it has to be 9 elements everywhere.

I'm guessing this will require changes anywhere the 8 is set by hand (which is potentially more places than just here)